src/cgnstools/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # cgnstools #
 #############
 
+add_definitions(-DCMAKE_BUILD)
```

Comment:
add_definition is not advisable in modern cmake. We should consider using target_compile_definition instead